recipes/openmmforcefields/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "openmmforcefields" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmmforcefields/archive/{{ version }}.tar.gz
+  sha256: af118a112de76f8b93c8ada8c55b4c12aa3deeb1164203aa3578fe7e266522e2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - openmm >=7.4.2
+    - pyyaml
+    - ambertools >=18.0
```

Comment:
Out of scope here, but is there a reason to not bump this to 20+?

Reply:
Oops, I misread the filename as `openforcefields` - feel free to disregard

Reply:
No clue. I just copied the reasonable entries from the existing recipes in Omnia. I recall this was something to do with ParmEd?

Reply:
Yep: https://github.com/omnia-md/conda-recipes/blob/master/openmmforcefields/meta.yaml#L64